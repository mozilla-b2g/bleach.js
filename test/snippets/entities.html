<p>&#x22;Something &amp; nothing&#x22; &Auml; &lt;script src="bad.js"&gt;&lt;&#47;script&#62;</p>