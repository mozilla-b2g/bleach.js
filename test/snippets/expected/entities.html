"Something & nothing" Ã„ <script src="bad.js"></script>